compression/zinfo.c

```diff
@@ -588,6 +593,18 @@ struct gzip_zinfo* blob_to_zinfo(void* buf, offset_t len)
     memcpy(&span_size, cur, 8);
     cur += 8;
 
+    claimed_size = PACKED_CHECKPOINT_SIZE * decode_int32(size) + BLOB_HEADER_SIZE;
+    if (claimed_size == len) {
+        // If we have exactly size checkpoints, then we have a current blob
```

Comment:
I'm think if `ZINFO_VERSION_CUR` is necessary. This might only work for `ZINFO_VERSION_TWO` given we might change something else breaking this in future `ZINFO_VERSION_THREE`, and we need to change it to `ZINFO_VERSION_TWO` (and comment `current` -> `v2`)